manifests/params.pp

```diff
@@ -5,21 +5,31 @@
 #
 class grafana::params {
   $archive_source      = undef
-  $cfg_location        = '/etc/grafana/grafana.ini'
   $cfg                 = {}
   $container_cfg       = false
   $container_params    = {}
   $data_dir            = '/var/lib/grafana'
   $docker_image        = 'grafana/grafana'
   $docker_ports        = '3000:3000'
   $install_dir         = '/usr/share/grafana'
-  $install_method      = 'package'
   $ldap_cfg            = false
-  $manage_package_repo = true
   $package_name        = 'grafana'
   $package_source      = undef
   $rpm_iteration       = '1'
-  $service_name        = 'grafana-server'
-  $version             = '2.5.0'
   $repo_name           = 'stable'
+  $version             = '2.5.0'
+  case $::osfamily {
+    'Archlinux': {
+      $manage_package_repo = false
+      $install_method      = 'repo'
```

Comment:
there is a package in the arch repo and thats almost always up2date, so we can rely on it.